### ISSUE TITLE
DocGen: CellContext of documentation files

### ISSUE THREAD

**Issue by kubaPod** [b3m2a1/mathematica-BTools#5]
A small suggestion for DocGen family.

Since the target group are third party packages maybe generated symbols etc. notebooks should have `CellContext -> Notebook` or even ``Global` ``, though Notebook will be better.

The reason is ``Needs["myPackage`"]`` in an intialization cell does not do much since every cell group has separate `$ContextPath`. One still needs to use full names or change `CellContext` themself, which will be confusing for newbies.

**Comment by b3m2a1** [comment]
@kubaPod good point. I haven't been setting anything outside of what the stylesheet does/is necessary to get the right functionality. I probably should add this if I'm going to include that ``Needs`` line. In the meantime you can use ``DocGen["SymbolPage", ..., CellContext->Notebook]`` to get this.
